src/ispc.cpp

```diff
@@ -1245,6 +1245,111 @@ Target::Target(Arch arch, const char *cpu, ISPCTarget ispc_target, PICLevel picL
     // Check default LLVM generated targets
     bool unsupported_target = false;
     switch (m_ispc_target) {
+    // TODO: figure out maskingFree and hasFeatures based on arch and CPU
```

Comment:
I do suggest putting a comment briefly defining what "generic" means, as it used to mean very different thing - the target that generated C++.

So something like "generic targets are the base targets that supposed to work on all hardware and serve as a baseline for implementing more fine-tuned targets"

Reply:
Done